RabbitMQ.pm

```diff
@@ -275,6 +293,12 @@ C<$props> is an optional hash (the AMQP 'props') respecting the following keys:
 
 =head2 consume($channel, $queuename, $options)
 
+Notify the RabbitMQ server that messages for the specified queue should
+be delivered on the specified channel.
+
+The consumer_tag is returned.  This command does B<not> return AMQP
+frames, for that the C<recv> method should be used.
```

Comment:
Fixed.  I also removed all mentions of frames, which do not seem important at this interface level.